README.md

```diff
@@ -6,6 +6,16 @@ sentry internal pypi
 this repository contains the tools to import and/or build packages from public pypi for the
 platforms and achitectures required for sentry development.
 
+## Install
+
+install dependencies before running any of the following scripts
+
+```bash
+brew install tox
+tox --devenv venv
+. venv/bin/activate
+```
+
```

Comment:
Personal preference is to get folks up and running as quickly as possible.

tox wasn't assumed knowledge for me and switching between sentry/getsentry, freight and this repo, they all install deps in a different way.

sentry/gentry: `pip install -r requirements-*.txt`
freight: `pip install . --index-url <internal pypi>`
pypi: `tox`

Expecting people to land in a repo, start eyeballing files to figure out what command to run when their goal it to add a package feels off to me.

That being said I'm a newbie to python AND ex-DevRel where a part of my role was to help with DX where possible, so this may be a warped perspective.

Reply:
yeah a lot of things are pretty different that's definitely true -- but I don't think we should duplicate the docs of external tools in every README (especially when they'll be prone to drift).  perhaps a `CONTRIBUTING.md` file or **`development`** readme section which simply says `Uses [tox]` and links to tox would be a decent compromise

coming from open source python though you'll basically either see tox or nox with the assumption that the tooling is known